docs/Makefile

```diff
@@ -18,7 +18,3 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
-
-test:
-	pytest 
-
```

Comment:
Remove it for now, I'll create a separate PR to create all these command in the appropriate directory.